packages/2018-neighborhood-development/src/components/VotersOnTheMove/index.js

```diff
@@ -1,25 +1,239 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { css } from 'emotion';
 
-import { CivicStoryCard, Placeholder } from '@hackoregon/component-library';
+import { CivicStoryCard, Scatterplot, Collapsable, CivicVictoryTheme, DataTable, ChartTitle } from '@hackoregon/component-library';
+
+import { fetchVotersOnTheMove, fetchAwayVotersOnTheMove } from '../../state/voters-on-the-move/actions';
+
+import {
+  isVotersOnTheMovePending,
+  catchVotersOnTheMoveErrors,
+  getVotersOnTheMoveData,
+  isAwayVotersOnTheMovePending,
+  catchAwayVotersOnTheMoveErrors,
+  getAwayVotersOnTheMoveData,
+} from '../../state/voters-on-the-move/selectors';
+
+const smallMultiples = css`
+  display: flex;
+  justify-content: center;
+  width: 100%;
+`;
+
+const chartColumn = css`
+  width: 100%
+`;
+
+const ageGroupLabels = [
+  { category: 18, label: '18-25' },
+  { category: 26, label: '26-32' },
+  { category: 33, label: '33-39' },
+  { category: 40, label: '40-49' },
+  { category: 50, label: '50+' },
+];
+
+const ageLabels = {
+  18: { category: 0, label: '18-25' },
+  26: { category: 1, label: '26-32' },
+  33: { category: 2, label: '33-39' },
+  40: { category: 3, label: '40-49' },
+  50: { category: 4, label: '50+' },
+};
+
+const legendStyle = css`
+  font-family: 'Roboto Condensed', 'Helvetica Neue', Helvetica, sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  text-align: center;
+  margin: 10px 0 0 0;
+`;
+
+const emphasis = css`
+  color: #000;
+`;
+
+const tableRows = [
+  { age_group: '18-25', towards: '50.3%', away: '49.7%', moves: '1.93' },
+  { age_group: '26-32', towards: '43.3%', away: '56.7%', moves: '1.97' },
+  { age_group: '33-39', towards: '41.1%', away: '58.9%', moves: '1.72' },
+  { age_group: '40-49', towards: '44.1%', away: '55.9%', moves: '1.52' },
+  { age_group: '55+', towards: '45.7%', away: '54.3%', moves: '1.40' },
+];
+
+const tableCols = [
+  {
+    align: 'left',
+    header: 'Age Group',
+    key: 'age_group',
+  },
+  {
+    align: 'left',
+    header: 'Average Number of Moves',
+    key: 'moves',
+  },
+  {
+    align: 'left',
+    header: 'Moves Towards City Center',
+    key: 'towards',
+  },
+  {
+    align: 'left',
+    header: 'Moves Away From City Center',
+    key: 'away',
+  },
+];
+
+const tableData = {
+  columns: tableCols,
+  data: tableRows,
+};
 
 export class VotersOnTheMove extends React.Component {
   componentDidMount() {
-    // initialize data here
+    this.props.init();
   }
 
   render() {
+    const {
+      isLoading,
+      error,
+      votersOnTheMove,
+      isAwayLoading,
+      awayError,
+      awayVotersOnTheMove,
+    } = this.props;
+
+    const data = { center: votersOnTheMove, away: awayVotersOnTheMove };
+
+    function legendComponent(age) {
```

Comment:
This is just a style thing but I feel like we should move this out of the `render` method. We could simplify the function signature to something like `const legendComponent = (age) => (lengendData) => ...` just to make it clear that we're currying here.

Reply:
Actually had the same thought when I was looking at this code earlier... So now that there's two voices saying it's weird I think I'm obligated to fix it 😝

Reply:
Actually had the same thought when I was looking at this code earlier... So now that there's two voices saying it's weird I think I'm obligated to fix it